internal/pkg/aws/ecs/ecs.go

```diff
@@ -229,7 +230,7 @@ func (s *Service) ServiceStatus() ServiceStatus {
 		Status:           aws.StringValue(s.Status),
 		DesiredCount:     aws.Int64Value(s.DesiredCount),
 		RunningCount:     aws.Int64Value(s.RunningCount),
-		LastDeploymentAt: s.Deployments[0].UpdatedAt.Unix(),
+		LastDeploymentAt: *s.Deployments[0].UpdatedAt, // FIXME Service assumed to have at least one deployment
```

Comment:
Do we an issue for this? would you mind creating it so we don't forget about it 🙏